openforcefield/tests/test_molecule.py

```diff
@@ -416,6 +417,18 @@ def test_serialization_no_conformers(self):
         pickle_copy = pickle.loads(pickle.dumps(mol))
         assert mol == pickle_copy
 
+    def test_json_numpy_roundtrips(self):
```

Comment:
👍  This is a cool test. I've always been vaguely concerned about precision loss and compounding rounding errors, so it's really neat to see this.

Reply:
I was pleasantly surprised to see it work